STAR/draft-ietf-acme-star.md

```diff
@@ -567,53 +566,52 @@ The rules to determine the notBefore and notAfter values of the i-th STAR
 certificate are as follows:
 
 ~~~
-    notBefore = nrd[i] - predating
-    notAfter  = min(nrd[i] + rcv, red)
+    notAfter  = min(nrd[i] + T, end)
+    notBefore = nrd[i] - max(adjust_client, adjust_server)
 ~~~
 
-where "predating" is the max between the (optional)
-recurrent-certificate-predate (rcp) and the amount of pre-dating that the
-server needs to add to make sure that all certificates being published are
-valid at the time of publication ({{fetching-certificates}}).  The server
-pre-dating is a fraction f of rcv (i.e., f * rcv with .5 <= f < 1).
+Where "adjust_client" is the min between the auto-renewal lifetime-adjust value
+("la"), optionally supplied by the client, and the auto-renewal lifetime of
+each short-term certificate ("T"); "adjust_server" is the amount of padding
+added by the ACME server to make sure that all certificates being published are
+valid at the time of publication.  The server padding is a fraction f of T
+(i.e., f * T with .5 <= f < 1, see {{fetching-certificates}}):
 
 ~~~
-    predating = max(rcp, f * rcv)
+    adjust_client = min(T, la)
+    adjust_server = f * T
 ~~~
 
+Note that the ACME server MUST NOT set the notBefore of the first STAR
+certificate to a date prior to the auto-renewal start-date.
+
 ### Example
 
 Given a server that intends to publish the next STAR certificate halfway
 through the lifetime of the previous one, and a STAR Order with the following
 attributes:
 
 ~~~
-     {
-       "recurrent-start-date": "2016-01-10T00:00:00Z",
-       "recurrent-end-date": "2016-01-20T00:00:00Z",
-       "recurrent-certificate-validity": 345600,    // 4 days
-       "recurrent-certificate-predate": 518400      // 6 days
+     "auto-renewal": {
+       "start-date": "2016-01-10T00:00:00Z",
+       "end-date": "2016-01-20T00:00:00Z",
+       "lifetime": 345600,          // 4 days
+       "lifetime-adjust": 259200    // 3 days
      }
 ~~~
 
-The amount of pre-dating that needs to be subtracted from each nominal renewal
-date is 6 days -- i.e., max(518400, 345600 * .5).
+The amount of time that needs to be subtracted from each nominal renewal
+date is 3 days -- i.e., max(min(345600, 259200), 345600 * .5).
 
 The notBefore and notAfter of each short-term certificate are:
 
 | notBefore | notAfter |
```

Comment:
Two of the time values in the table are misformatted.

Reply:
good catch, thanks!